tests/simulator/initialize/test_init_mhd.py

```
@@ -150,3 +295,321 @@ def p(*xyz):
+                                )
+                            ]
+                            for d in range(dim)
+                        ]
+                    mesh = [a.flatten() for a in np.meshgrid(*coords, indexing="ij")]
+                    exp = np.asarray(fns[ci](*mesh)) + np.zeros(data.size)
+                    assert_fp_any_all_close(
+                        data.flatten(), exp.flatten(), atol=atol, rtol=0
+                    )
+
+    def _assert_mhd_div_free(self, hier, prefix, dim, atol):
+        for ilvl, level in hier.levels().items():
```

Ruff (0.15.20)

[warning] Loop control variable `ilvl` not used within loop body

Rename unused `ilvl` to `_ilvl`

(B007)